.pre-commit-config.yaml

```diff
@@ -1,12 +1,12 @@
 ---
 repos:
   - repo: git://github.com/Lucas-C/pre-commit-hooks
-    rev: v1.1.9
+    rev: v1.1.13
     hooks:
       - id: remove-tabs
 
   - repo: git://github.com/pre-commit/pre-commit-hooks
-    rev: v3.4.0
+    rev: v4.1.0
```

Comment:
same here (switch from `git://` to `https://`), but also there is another `pre-commit/pre-commit-hooks` section below, so better fold the hooks into that one (and remove duplicates)